Cargo.toml

```diff
@@ -82,13 +86,17 @@ ibc-core-commitment-types   = { version = "0.56", default-features = false }
 ibc-core-handler-types      = { version = "0.56", default-features = false }
 ibc-client-tendermint-types = { version = "0.56", default-features = false }
 
-alloy                  = { version = "0.8", default-features = false }
```

Comment:
We can discuss this, but I just felt it was a bit much to depend on the whole thing, when we can just pick and choose the bits we need.

Reply:
Ok, I did that now, but I am not super happy about how it is turning out, since we will need most of the alloy-dependencies separately anyways. Now we end up with double up of everything here, which seems messier and harder to deal with version-wise than just having each of them separately 🤷 

Like I said earlier, I am OK with it, but I personally think it is cleaner and more neater to just pull exactly what we need. I don't like having all these extra features and whatnot which we don't even use. I would expect code-examples to use it, because it makes them smaller, but for production applications I wouldn't necessarily prefer it.